cmd/beacon-chain/jwt/jwt.go

```diff
@@ -0,0 +1,82 @@
+package jwt
+
+import (
+	"errors"
+	"fmt"
+
+	"path/filepath"
+
+	"github.com/prysmaticlabs/prysm/crypto/rand"
+	"github.com/prysmaticlabs/prysm/io/file"
+
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/prysmaticlabs/prysm/cmd"
+	"github.com/prysmaticlabs/prysm/runtime/tos"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+)
+
+var log = logrus.WithField("prefix", "jwt")
+
+var Commands = &cli.Command{
+	Name:        "generate-jwt-secret",
+	Usage:       "creates a random 32 byte hex string in a plaintext file to be used for authenticating JSON-RPC requests. If no --output-file flag is defined, the file will be created in the current working directory",
+	Description: `creates a random 32 byte hex string in a plaintext file to be used for authenticating JSON-RPC requests. If no --output-file flag is defined, the file will be created in the current working directory`,
+	Flags: cmd.WrapFlags([]cli.Flag{
+		cmd.JwtOutputFileFlag,
+	}),
+	Before: tos.VerifyTosAcceptedOrPrompt,
+	Action: func(cliCtx *cli.Context) error {
+		specifiedFilePath := cliCtx.String(cmd.JwtOutputFileFlag.Name)
+		if err := generateHttpSecretInFile(specifiedFilePath); err != nil {
+			log.Printf("Could not generate secret: %v", err)
+		}
+		return nil
+	},
+}
+
+func generateHttpSecretInFile(specifiedFilePath string) error {
```

Comment:
Let's add a unit test to make sure this function works as expected

Reply:
Will do - I'll also test this IRL with geth to make sure everything works before marking as ready for review.